### PR TITLE
removed redundancy and improved readability

### PR DESCRIPTION
**Pull request by carloscaldas**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/685)

### CODE REVIEW

**Comment by carloscaldas** [comment]
test!

File: src/main/scala/beam/calibration/utils/DeleteSuggestion.scala [hunk 1]
```diff
@@ -31,55 +31,44 @@ object DeleteSuggestion extends LazyLogging {
...
   def listSuggestions(experimentId: String) = {
     BeamSigoptTuner.fetchExperiment(experimentId) match {
-      case Some(_experiment) => {
-
-        if (_experiment.suggestions().list().call().getData.size() > 0)
-          _experiment.suggestions().list().call().getData.forEach { d =>
-            println(d)
-          } else
+      case Some(_experiment) =>
+        val data = _experiment.suggestions().list().call().getData
```
Comment (JustinPihony) [inline src/main/scala/beam/calibration/utils/DeleteSuggestion.scala]: Nice catch! This isn't just redundancy of verbiage, but also work.

File: src/main/scala/beam/calibration/utils/DeleteSuggestion.scala [hunk 2]
```diff
@@ -31,55 +31,44 @@ object DeleteSuggestion extends LazyLogging {
...
 
-  def deleteAllOpenSuggestions(experimentId: String) = {
-
+  def deleteAllOpenSuggestions(experimentId: String): Unit = {
     BeamSigoptTuner.fetchExperiment(experimentId) match {
-      case Some(_experiment) => {
-
-        if (_experiment.suggestions().list().call().getData.size() > 0)
+      case Some(_experiment) =>
+        if (!_experiment.suggestions().list().call().getData.isEmpty) {
```
Comment (JustinPihony) [inline src/main/scala/beam/calibration/utils/DeleteSuggestion.scala]: Can you do the same cleanup here? Store the redundant `getData` into a top level variable?

File: src/main/scala/beam/calibration/utils/DeleteSuggestion.scala [hunk 3]
```diff
@@ -28,68 +27,55 @@ object DeleteSuggestion extends LazyLogging {
...
-          _experiment.suggestions().list().call().getData.forEach { d =>
-            println(d)
-          } else
-          logger.info(s"Experiement with id $experimentId has no suggestion")
-      }
-      case None => {
+      case Some(_experiment) =>
+        val data = _experiment.suggestions().list().call().getData
+        data.forEach(println)
+        if (data.isEmpty) {
```
Comment (JustinPihony) [inline src/main/scala/beam/calibration/utils/DeleteSuggestion.scala]: I see what you did here, but for semantics, it still seems that it should be `if is empty log else print each` I'm fine either way.

File: src/main/scala/beam/calibration/utils/DeleteSuggestion.scala [hunk 4]
```diff
@@ -28,68 +27,55 @@ object DeleteSuggestion extends LazyLogging {
...
-          } else
-          logger.info(s"Experiement with id $experimentId has no suggestion")
-      }
-      case None => {
+      case Some(_experiment) =>
+        val data = _experiment.suggestions().list().call().getData
+        if (data.isEmpty) {
+          logger.info(s"Experiment with id $experimentId has no suggestion")
+        }
+        data.forEach { d =>
```
Comment (JustinPihony) [inline src/main/scala/beam/calibration/utils/DeleteSuggestion.scala]: Again, possibly add the else for semantics - but not really required and more aesthetics.

**Pull request merged**